primitives/npos-elections/src/helpers.rs

```diff
@@ -75,6 +78,28 @@ pub fn assignment_staked_to_ratio_normalized<A: IdentifierT, P: PerThing128>(
 	Ok(ratio)
 }
 
+/// This can only fail of the normalization fails. This can happen if for any of the resulting
+/// assignments, `assignment.distribution.map(|p| p.deconstruct()).sum()` fails to fit inside
+/// `UpperOf<P>`. A user of this crate may statically assert that this can never happen and safely
+/// `expect` this to return `Ok`.
+pub(crate) fn voter_candidate_to_election_result<AccountId: IdentifierT, P: PerThing128>(
```

Comment:
```suggestion
#[cfg(test)]
pub(crate) fn voter_candidate_to_election_result<AccountId: IdentifierT, P: PerThing128>(
```

Reply:
and if so, it should go to the mock.rs file.

Reply:
This is not needed for this PR, it will be for the `MMS` one as it reuses that piece of code. Removing it from this one.